certificate_agent.py

```diff
@@ -120,11 +120,12 @@ def main():
                     name=name,
                     course_id=course_id,
                     grade=grade,
+                    designation=designation,
                 )
             )
             (download_uuid,
              verify_uuid,
-             download_url) = cert.create_and_upload(name, grade=grade)
+             download_url) = cert.create_and_upload(name, grade=grade, designation=designation)
```

Comment:
Now we're passing `designation` to `create_and_upload`, but you didn't actually update `create_and_upload` to accept this `kwarg`.